static/app/components/compactSelect/control.tsx

```diff
@@ -576,36 +601,76 @@ const ClearButton = styled(Button)`
   margin: -${space(0.25)} -${space(0.5)};
 `;
 
+const searchCursorBlink = keyframes`
```

Comment:
This mock cursor looks identical on macos, but is that the case for other os/browsers? Worried that it might look out of place in some circumstances?

Reply:
I also noticed that the blinking cursor doesn't show up if you have some text inside the searchbox and hover over an option. So there is some slight inconsistency in the behavior 😞 

Still seems like the best option would be to support virtual focus in the grid list, but this is an improvement over what we have now.

Reply:
The blinking cursor in Windows (10) looks the same:
<img width="412" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/78246558-040b-4ec5-b9cd-a5c20e93a0b3">

There's a bigger difference in iOS/iPadOS, but the switch between the two cursors doesn't seem too jarring imo, it just looks like the text box just received focus:

https://github.com/getsentry/sentry/assets/44172267/22ce3499-e3a6-4e79-a32e-6488cd46994e

Reply:
Honestly I think we should just drop the cursor.

This is one of those nice features where when it "just works" people are happy. For those who aren't UI power users it's fine, they can just click. 

IMO we're going too out of the way here to have an affordance for this with a lot of extra code

Reply:
Yeah I agree that the best solution is virtual focus support in grid list. That doesn't seem to be forthcoming, given the way that `useGridList()` is being implemented. 

Specifically, in grid widgets users should be able to use the Arrow Left/Right keys to move focus horizontally across cells in the grid. In the project selector, those cells are the trailing buttons/links:

<img width="392" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/73c8530a-03b3-439e-8379-ae2a90bd9ff0">

While `react-aria` can simulate virtual focus on the options themselves (since the options are implemented with `useGridListItem()`), it can't do the same thing to the trailing buttons/links — those elements are not directly hooked up to `react-aria`. What `react-aria` can do is [pass real focus to the next/previous focusable element](https://github.com/adobe/react-spectrum/blob/8564df0fec1775345eb357648a98160ab14129ff/packages/%40react-aria/gridlist/src/useGridListItem.ts#L85-L95). To fully implement virtual focus, we'll need special hooks for each of the trailing buttons/links (something like `useGridListCell()`).